static/js/stream_list.js

```diff
@@ -218,7 +218,23 @@ exports.set_in_home_view = function (stream_id, in_home) {
 
 function build_stream_sidebar_li(sub) {
     var name = sub.name;
+    var description = stream_data.get_sub_by_name(name).description;
+
+    // WIP; generates wrong, also shouldn't be here (descriptions have to be updated)
+    channel.post({
+        url: '/json/messages/render',
+        idempotent: true,
+        data: {content: description},
+        success: function (response_data) {
+            description = response_data.rendered;
+        },
+        error: function () {
+            description = i18n.t("Failed to generate markdown");
+        },
+    });
```

Comment:
would `markdown.js` require refactoring, as it seems it handles message-format items only right now?